OpenOversight/tests/conftest.py

```diff
@@ -315,16 +315,16 @@ def db(app):
 def session(db):
     """Creates a new database session for a test."""
     connection = db.engine.connect()
```

Comment:
Optional but we could use a context manager for this https://docs.sqlalchemy.org/en/13/core/connections.html#using-transactions

Reply:
If you feel like that'd be clearer, I'm here for it! I'll make the modification sometime tomorrow.

Reply:
Done.